kong/runloop/handler.lua

```diff
@@ -540,6 +569,7 @@ local function _set_update_plugins_iterator(f)
   update_plugins_iterator = f
 end
 
+-- TODO: how can we reset counters if kong is reconfigured
```

Comment:
Do we have any progress on it?

Reply:
> Do we have any progress on it?

Based on datong's opinion, we don't need to reset it. Because the role of ready endpoint is to make sure the first configuration is ok. Whatever happens next is not in its consideration.